scio-repl/src/main/scala/com/spotify/scio/repl/ScioILoop.scala

```diff
@@ -89,7 +90,7 @@ class ScioILoop(scioClassLoader: ScioReplClassLoader,
     val sc = if (name.nonEmpty) name else "sc"
     val rsc = "com.spotify.scio.repl.ReplScioContext"
     val opts = optsFromArgs(scioOpts)
-    val nextReplJar = scioClassLoader.getNextReplCodeJarPath
+    val nextReplJar = StringEscapeUtils.escapeJava(scioClassLoader.getNextReplCodeJarPath)
```

Comment:
@f-loris thanks for your PR!

instead of introducing a new dependency ... wouldn't the following work as well? 
```scala
val nextReplJar = s"""${scioClassLoader.getNextReplCodeJarPath}"""
```

Reply:
@regadas Thanks for your feedback. Unfortunately this doesn't work, otherwise it would have already worked as the line below adds `nextReplJar` inside a `"""` block. 

The result of `s"""${scioClassLoader.getNextReplCodeJarPath}"""` still looks like this `C:\Users\xy\` but it has to look like `"C:\\Users\\xy\\` since `nextReplJar` get interpreted as such. 

Without introducing a new dependency a simple solution would be replacing `\` with `\\`:

```scala
 val nextReplJar = scioClassLoader.getNextReplCodeJarPath.replace("\\","\\\\")
```

Reply:
@f-loris after a closer look i see now that the problem happens when the interpreter eval's the interpolated string. Let's keep `StringEscapeUtils.escapeJava`

Thanks!